### PR TITLE
Chemistry cleanup

### PR DESCRIPTION
**Pull request by fxcoudert**
This is a cleanup of the chemistry-related packages:

- adding tag and DOI to all
- adding head URL to `cp2k`
- updating `packmol` and adding head URL

Although `voro++` is not a chemistry package, as a regular user I also included fixes for it.

----

I have:

- [X] Followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew-science/blob/master/.github/CONTRIBUTING.md) document?
- [X] Checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-science/pulls) for the same update/change?
- [X] Run `brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting) and corrected all errors?
- [X] Built your formula locally prior to submission with `brew install <formula>`?
- [X] Not altered the `bottle` section?
- [X] Checked that the tests still pass?

### CODE REVIEW

File: packmol.rb [hunk 1]
```diff
@@ -1,8 +1,11 @@
 class Packmol < Formula
   desc "Packing optimization for molecular dynamics simulations"
   homepage "http://www.ime.unicamp.br/~martinez/packmol/"
-  url "http://leandro.iqm.unicamp.br/packmol/versionhistory/packmol-16.320.tar.gz"
-  sha256 "792df9f318189df19e9281237862c3de3cf3dc8e95c130b98ff107bf097a4243"
+  url "https://github.com/leandromartinez98/packmol/archive/16.344.tar.gz"
+  sha256 "09ad18c01f46f3b1afd645bba41adcc2870f14572c7c4148962ade221241a14a"
```
Comment (jonchang) [inline packmol.rb]: Since this is a new version and bottles will need to be rebuilt, please spin this off into a new pull request.

Comment (jonchang) [inline packmol.rb]: Since this is a new version and bottles will need to be rebuilt, please spin this off into a new pull request.

File: voro++.rb [hunk 2]
```diff
@@ -1,23 +1,26 @@
 class Voroxx < Formula
+  desc "3D Voronoi cell software library"
   homepage "http://math.lbl.gov/voro++"
   url "http://math.lbl.gov/voro++/download/dir/voro++-0.4.6.tar.gz"
   sha256 "ef7970071ee2ce3800daa8723649ca069dc4c71cc25f0f7d22552387f3ea437e"
   head "https://codeforge.lbl.gov/anonscm/voro/trunk", :using => :svn
+  # tag "math"
+  # doi "10.1063/1.3215722"
```
Comment (jonchang) [inline voro++.rb]: Since this doesn't have a bottle yet, please split this off into a separate pull request so it'll be easier to manage the bottles.

**Comment by fxcoudert** [comment]
@jonchang I'm learning git, so it was a good exercice :)

I have separated the voro++ and packmol changes, and edited the commit messages.

**Comment by jonchang** [comment]
Thanks @fxcoudert. The other pull requests will have to wait for a little bit for our CI backlog to clear.
